R/proportional_hazards-glmnet.R

```diff
@@ -520,6 +520,7 @@ get_missings_coxnet <- function(new_x, new_strata) {
 #' @param time Deprecated in favor of `eval_time`. A vector of integers for prediction times.
 #' @param output One of "surv" or "haz".
 #' @param penalty Penalty value(s).
+#' @param multi Allow multiple penalty values?
```

Comment:
Should mention the default

```suggestion
#' @param multi Allow multiple penalty values? Defaults to FALSE.
```